src/trace/traceManager.js

```diff
@@ -88,12 +88,11 @@ TraceManager.prototype.getStorageAt = function (stepIndex, tx, callback, address
   if (this.traceRetriever.debugStorageAtAvailable()) {
     this.traceRetriever.getStorage(tx, address, function (error, result) {
       if (error) {
-        console.log(error)
-        callback(error, null)
-      } else {
-        var storage = self.traceCache.rebuildStorage(address, result, stepIndex)
-        callback(null, storage)
+        // TODO throws proper error when debug_storageRangeAt will be available
+        result = {}
       }
+      var storage = self.traceCache.rebuildStorage(address, result, stepIndex)
```

Comment:
We could still log something (I pushed that). This is what we've done before.
Anyway there's  now not so much notification error in Remix.... that will be addressed by https://github.com/ethereum/remix/issues/199